### PR TITLE
refactor: remove dup of code from linter.js

### PR DESCRIPTION
**Pull request by Gabriel-Dias-Oliveira**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [X] **code refactor**
- [X] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

- It is removing an unnecessary function since it already exists on **utils.js**. 
- Creating some consts to avoid some object property repetition.

### Additional Info

This refactoring does not change any logic, it's just a suggestion for refactoring, and feels free to comment!

### CODE REVIEW

**Review by alexander-akait — DISMISSED** [review]
/cc @ricardogobbosouza

File: src/utils.js [hunk 1]
```diff
@@ -89,6 +89,14 @@ function parseFoldersToGlobs(patterns, extensions = []) {
   });
 }
 
+/**
+ * @param {any[]} list
+ * @returns {boolean}
+ */
+function hasElementsOn(list = []) {
+  return list.length > 0;
+}
```
Comment (ricardogobbosouza) [inline src/utils.js:98]: Doesn't need a function to check the length

**Review by ricardogobbosouza — CHANGES_REQUESTED** [review]
Just remembering that this is a totally opinionated refactoring

File: src/utils.js (hunk 1 above)
Comment (ricardogobbosouza) [reply src/utils.js:98]: Because it's just a length check, it doesn't make sense.

Comment (ricardogobbosouza) [reply src/utils.js:98]: I think that's all, the rest looks good.
Thanks @Gabriel-Dias-Oliveira

Comment (Gabriel-Dias-Oliveira) [reply src/utils.js]: Agree :)
It should be fixed now.

**Review by ricardogobbosouza — APPROVED** [review]

**Pull request merged**